### ISSUE TITLE
`Canvas.toDataURL` doesn't work

### ISSUE THREAD

**Issue by RuiNtD** [DjDeveloperr/skia_canvas#62]
`Canvas.toDataURL` doesn't work, returning this error:

```
error: Uncaught (in promise) TypeError: encodeBase64 is not a function
    return `data:image/${format};base64,${encodeBase64(buffer)}`;
                                          ^
    at Canvas.toDataURL (https://deno.land/x/skia_canvas@0.5.5/src/canvas.ts:151:43)
```

Using deno 1.40.1

**Comment by RuiNtD** [comment]
This also affects setting an `Image`'s `src` to a data URL:
```
error: Uncaught (in promise) TypeError: decodeBase64 is not a function
          ? decodeBase64(content)
            ^
    at Image.set src (https://deno.land/x/skia_canvas@0.5.5/src/image.ts:75:13)
```

**Comment by DjDeveloperr** [comment]
This should be fixed in 0.5.6
